safe-core-api/service-architecture.md

```diff
@@ -1,34 +1,36 @@
 ---
-description: Up-to-date overview document on how our services interact with each other
+description: An overview of how our services interact with each other
 ---
 
 # Service architecture
 
-The Safe infrastructure consists of 4 services:
+The Safe infrastructure consists of four services:
 
-* [Safe Transaction Service](https://github.com/safe-global/safe-transaction-service): Keeps track of transactions related to Safe contracts (Python)
+* [Safe Transaction Service](https://github.com/safe-global/safe-transaction-service): Keeps track of transactions related to Safe contracts (Python).
 * [Safe Events Service](https://github.com/safe-global/safe-events-service): Handles Safe indexing events from the Transaction Service and delivers them as HTTP webhooks.
-* [Safe Config Service](https://github.com/safe-global/safe-config-service): Keeps track of all supported networks and chain-specific variables (Python)
-* [Safe Client Gateway](https://github.com/safe-global/safe-client-gateway-nest): Uses the config service to determine how to reach the transaction service instance required for a given request (Node.js)
+* [Safe Config Service](https://github.com/safe-global/safe-config-service): Keeps track of all supported networks and chain-specific variables (Python). Also referred to as *config service*.
```

Comment:
It makes sense that we remove "Safe" when referencing our services sometimes but I think we should always use cammel case. I'm specifically talking about *config service* and *client gateway*. Wdyt?